pom.xml

```diff
@@ -289,8 +289,8 @@ Import-Package: \\
               <classpath>${project.build.directory}/dependency</classpath>
             </compilerArguments>
             <compilerArgs>
-              <arg>-err:+nullAnnot(org.eclipse.jdt.annotation.Nullable|org.eclipse.jdt.annotation.NonNull|org.eclipse.jdt.annotation.NonNullByDefault),+inheritNullAnnot,-nullUncheckedConversion</arg>
-              <arg>-warn:+null,+inheritNullAnnot,+nullAnnotConflict,-nullUncheckedConversion,+nullAnnotRedundant,+nullDereference</arg>
+              <arg>-err:+nullAnnot(org.eclipse.jdt.annotation.Nullable|org.eclipse.jdt.annotation.NonNull|org.eclipse.jdt.annotation.NonNullByDefault),+inheritNullAnnot,+nullAnnotConflict,-nullUncheckedConversion</arg>
+              <arg>-warn:+null,+inheritNullAnnot,-nullUncheckedConversion,+nullAnnotRedundant,+nullDereference</arg>
```

Comment:
@wborn, do you think that having `+inheritNullAnnot` in `-err` AND `-warn` is correct? I would assume that it should be added only in one place.

Reply:
I also found it strange the errors arguments are also part of the warnings. But I didn't look into if there is also some kind of option where the compiler continues on errors but still logs these problems on warnings where such configuration might make sense. If this mode doesn't exist or not work this way I think they can be removed from the warnings. :-)